src/py42/sdk/__init__.py

```diff
@@ -134,25 +163,121 @@ def securitydata(self):
+        kv_service, alert_rules_key, auth=main_auth
+    )
+    alerts_conn = Connection.from_microservice_key(
+        kv_service, alerts_key, auth=main_auth
+    )
+    file_events_conn = Connection.from_microservice_key(
+        kv_service, file_events_key, auth=main_auth
+    )
+    pds_conn = Connection.from_microservice_key(
+        kv_service, preservation_data_key, auth=main_auth
+    )
+    ecm_conn = Connection.from_microservice_key(
+        kv_service, employee_case_mgmt_key, auth=main_auth
+    )
+
+    user_svc = UserService(main_connection)
+    administration_svc = AdministrationService(main_connection)
+    file_events_service = FileEventService(file_events_conn)
+    user_ctx = UserContext(administration_svc)
+    user_profile_svc = DetectionListUserService(ecm_conn, user_ctx, user_svc)
+
+    services = Services(
+        administration=administration_svc,
+        archive=ArchiveService(main_connection),
+        devices=DeviceService(main_connection),
+        legalhold=LegalHoldService(main_connection),
+        orgs=OrgService(main_connection),
+        securitydata=SecurityDataService(main_connection),
+        users=UserService(main_connection),
+        alertrules=AlertRulesService(alert_rules_conn, user_ctx, user_profile_svc),
+        alerts=AlertService(alerts_conn, user_ctx),
+        fileevents=file_events_service,
+        savedsearch=SavedSearchService(file_events_conn, file_events_service),
+        preservationdata=PreservationDataService(pds_conn),
+        departingemployee=DepartingEmployeeService(
+            ecm_conn, user_ctx, user_profile_svc
+        ),
+        highriskemployee=HighRiskEmployeeService(ecm_conn, user_ctx, user_profile_svc),
+        userprofile=user_profile_svc,
+    )
+
+    return services, user_ctx
+
+
+def _init_clients(services, connection):
+    authority = AuthorityClient(
+        administration=services.administration,
+        archive=services.archive,
+        devices=services.devices,
+        legalhold=services.legalhold,
+        orgs=services.orgs,
+        securitydata=services.securitydata,
+        users=services.users,
+    )
+    detectionlists = DetectionListsClient(
+        services.userprofile, services.departingemployee, services.highriskemployee
+    )
+
+    storage_service_factory = StorageServiceFactory(
+        connection, services.devices, ConnectionManager()
+    )
+    alertrules = AlertRulesClient(services.alerts, services.alertrules)
+    securitydata = SecurityDataClient(
+        services.securitydata,
+        services.fileevents,
+        services.preservationdata,
+        services.savedsearch,
+        storage_service_factory,
+    )
+    alerts = AlertsClient(services.alerts, alertrules)
+    archive_accessor_mgr = ArchiveAccessorManager(
+        services.archive, storage_service_factory
+    )
+    archive = ArchiveClient(archive_accessor_mgr, services.archive)
+    clients = Clients(
```

Comment:
This is a different order than defined in `clients/__init__.py`, causing nothin to work for me